### PR TITLE
completed P0 - home page

### PR DESCRIPTION
**Pull request by vidhi-thakur**
home page of the e-commerce project, it has 
- `out of stock` text overlay card
- one responsive image
- horizontal cards with badge

**Screenshots: -**
image
image

### CODE REVIEW

**Review by ravikumar0403 — APPROVED** [review]
Looks good to me

File: index.html [hunk 1]
```diff
@@ -1,12 +1,169 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
```
Comment (yashpatel1797) [inline index.html:7]: Not Mandatory but you can add an author tag.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to me.

File: styles/home.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+@media screen and (max-width: 1290px) {
+  .card-container {
+    flex-wrap: wrap;
+    margin: 0 3.125rem;
+  }
+}
+
+a {
```
Comment (msdeshmukh009) [inline styles/home.css]: It's not a big issue but styling the elements directly is not really a good practice so you can consider making class for this styles.

**Review by msdeshmukh009 — APPROVED** [review]
Rest looks good to me.

File: styles/home.css (hunk 2 above)
Comment (vidhi-thakur) [reply styles/home.css]: Thank you for pointing this out

File: index.html (hunk 1 above)
Comment (vidhi-thakur) [reply index.html:7]: thanks for pointing out, will update

**Pull request merged**